src/eko/io/legacy.py

```diff
@@ -92,10 +100,16 @@ def from_old(cls, old: RawCard):
         """Load from old metadata."""
         heavy = HeavyInfo(
             num_flavs_init=4,
-            num_flavs_max_pdf=None,
-            intrinsic_flavors=None,
-            masses=HeavyQuarkMasses([1.51, 4.92, 172.5]),
-            masses_scheme=None,
+            num_flavs_max_pdf=5,
```

Comment:
On a similar point can we start dropping this?

Reply:
Same as above

Reply:
#295